recipe/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "rbcl" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a0bf43234a2987b67b451d1e61efe5666c12a236001e8ace788307ce5f2ad698
+  patches:
+    - 0001-do-not-compile-binary.patch
+    - 0002-import-appropriate-libsodium-binary-depending-on-sys.patch
+
+build:
+  number: 0
+  script:
+    - mkdir -p $SRC_DIR/build/temp/lib  # [not win]
+    - mkdir -p %SRC_DIR%\build\temp\lib  # [win]
+    - cp $PREFIX/lib/libsodium.* $SRC_DIR/build/temp/lib  # [not win]
```

Comment:
is it really necessary, do not we have in env already ?

Reply:
Is there a specific preference as to using `os.environ['PREFIX']` inside the install files? I usually try to touch the install files as little as possible in favor of doing changes in the script section.

Reply:
fixed.